src/proxy/http/profiles.rs

```diff
@@ -16,7 +20,13 @@ use never::Never;
 use super::retry::Budget;
 use NameAddr;
 
-pub type Routes = Vec<(RequestMatch, Route)>;
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct ConcreteDst(pub NameAddr);
```

Comment:
Nit: I don't find this name terribly obvious, but I can't come up with any better suggestions OTTOMH. To me, `concrete` doesn't really convey "final destination after applying weighting" but I'm not sure what would.